### PR TITLE
Cleanup stuff that's not relevant for AdGear

### PR DESCRIPTION
**Pull request by frankgrimes97**

### CODE REVIEW

**Comment by jbfavre** [comment]
Are you serious ?

File: Dockerfile.centos.7_9.x [hunk 1]
```diff
@@ -1,5 +1,4 @@
 FROM centos:centos7
-MAINTAINER Jean Baptiste Favre <docker@jbfavre.org>
```
Comment (jbfavre) [inline Dockerfile.centos.7_9.x:2]: you dare calling this a minor changes ?
How does "stealing other's work" looks like to you ?

**Comment by frankgrimes97** [comment]
Apologies, I believe this PR was created by mistake. Yes, we forked it for our own internal usage.

File: Dockerfile.centos.7_9.x (hunk 1 above)
Comment (frankgrimes97) [reply Dockerfile.centos.7_9.x:2]: I assure you this was not our intention.
How best can we resolve this?
I can certainly put you back as MAINTAINER in our fork... it just didn't seem appropriate for our developers to contact you for something we plan to maintain ourselves.

**Comment by frankgrimes97** [comment]
Closing this pull request because it was created accidentally. The changes in our fork were never intended to be incorporated upstream.

**Pull request closed without merging**